apis/infix/atrium-api-infix/src/commonTest/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/FeatureExtractorSamples.kt

```diff
@@ -1,34 +1,456 @@
 package ch.tutteli.atrium.api.infix.en_GB.samples
 
-import ch.tutteli.atrium.api.infix.en_GB.*
+import ch.tutteli.atrium.api.infix.en_GB.because
+import ch.tutteli.atrium.api.infix.en_GB.extractSubject
+import ch.tutteli.atrium.api.infix.en_GB.feature
+import ch.tutteli.atrium.api.infix.en_GB.its
+import ch.tutteli.atrium.api.infix.en_GB.notToHaveElementsOrAll
+import ch.tutteli.atrium.api.infix.en_GB.of
+import ch.tutteli.atrium.api.infix.en_GB.toBeGreaterThan
+import ch.tutteli.atrium.api.infix.en_GB.toBeLessThan
+import ch.tutteli.atrium.api.infix.en_GB.toContain
+import ch.tutteli.atrium.api.infix.en_GB.toEndWith
+import ch.tutteli.atrium.api.infix.en_GB.toHaveElementsAndAll
+import ch.tutteli.atrium.api.infix.en_GB.toStartWith
+import ch.tutteli.atrium.api.infix.en_GB.withFailureDescription
 import ch.tutteli.atrium.api.verbs.expect
 import kotlin.test.Test
 
 class FeatureExtractorSamples {
+
     data class Person(
-        val firstName: String,
-        val lastName: String,
-        val age: Int,
-        val children: Collection<Person>
+        val name: String, val age: Int
+    )
+
+    @Test
+    fun itsFeature() {
+        val person = Person(name = "John Smith", age = 25)
+
+        expect(person) its { age } toBeLessThan 30 toBeGreaterThan 20 // subject is now Int
+
+
+        fails {
+            expect(person) its { age } toBeLessThan 20 toBeGreaterThan 30 // subject is now Int; fails
+        }
+    }
+
+    @Test
+    fun its() {
+        val person = Person(name = "John Smith", age = 25)
+
+        expect(person) its feature({ age }) {
+            toBeGreaterThan(18)
+            toBeLessThan(35)
```

Comment:
use `it ..` (also further below)
```suggestion
            it toBeGreaterThan 18
            it toBeLessThan 35
```

Reply:
Done 👍